pkgs/by-name/mk/mkbootimage/package.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, libelf
+, pcre
+}:
+
+stdenv.mkDerivation {
+  pname = "mkbootimage";
+  version = "2.3-unstable-2022-05-26";
+
+  src = fetchFromGitHub {
+    owner = "antmicro";
+    repo = "zynq-mkbootimage";
+    rev = "872363ce32c249f8278cf107bc6d3bdeb38d849f";
+    hash = "sha256-5FPyAhUWZDwHbqmp9J2ZXTmjaXPz+dzrJMolaNwADHs=";
+  };
+
+  buildInputs = [ libelf pcre ];
+
+  postPatch = ''
+    substituteInPlace Makefile --replace "git rev-parse --short HEAD" "echo nix"
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm755 mkbootimage -t $out/bin
+    runHook postInstall
+  '';
```

Comment:
```suggestion
  installPhase = ''
    runHook preInstall
 
    install -Dm755 mkbootimage -t $out/bin
 
    runHook postInstall
  '';
```

Reply:
Decide between you and @quantenzitrone what is the unwritten rule about nitpicking.https://github.com/NixOS/nixpkgs/pull/267244#issuecomment-1836199851

> I think you shouldn't enforce any formatting rules (except for incredibly hideous formatting) until RFC 166 gets accepted (and thus doesn't get any more changes without a new RFC). At the current stage of RFC 166, there is still a lot of discussion on specific formatting rules.

Reply:
I've heard of this convention, but is it written down somewhere? If not, I'd say nitpick.
I can't find it in [pkgs/README.md](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md) or the Nixpkgs Manual.
Furthermore, I have found example code in the Nixpkgs Manual that does not follow this convention: https://github.com/NixOS/nixpkgs/blob/631eca2e965325b4f006ae871421baa0b0074c79/doc/languages-frameworks/texlive.section.md?plain=1#L84-L94